Makefile

```diff
@@ -74,33 +75,44 @@ tests/test_playbooks/vars/server.yml:
 	@echo "For recording, please adjust $@ to match your reference server."
 
 dist-test: $(MANIFEST)
-	ANSIBLE_COLLECTIONS_PATHS=build/collections ansible -m $(NAMESPACE).$(NAME).organization -a "username=admin password=changeme server_url=https://foreman.example.test name=collectiontest" localhost | grep -q "Failed to connect to Foreman server"
```

Comment:
why did you remove that?

Reply:
That is a test, that our modules (pars pro toto) work as a collection.
We now test all the modules as a collection. So it felt duplicate.
We do not have anything similar for the docs (yet) however, and i left that.

Reply:
ah. I actually think that's still usefull, as it makes it easier to test "did the collection build fine" w/o running the testsuite.